locales/ca/generate.json

```diff
@@ -0,0 +1,52 @@
+{
+   "meta": {
+      "description": "Genera llistes de música a Spotify basat en les cançons que estàn tocant en aquell tour, utilizant la API de Setlist.fm",
+      "keywords": "Spotify, Setlist.fm, Generador de llistes, Festivals, Música, Concerts",
+      "title": "Generar | Festwrap"
+   },
+   "steps": {
+      "navigation": {
+         "finish": "Finalitzar",
+         "generate": "Generar",
+         "next": "Següent",
+         "previous": "Anterior"
+      },
+      "step1": {
+         "description": "Selecciona per crear una nova llista o utilitza una llista ja existent en el teu compte",
+         "form": {
+            "createNewPlaylist": {
+               "description": "Utilitza aquesta opció per crear una nova llista de reproducció a Spotify",
+               "giveAName": "Dóna-li un nom a la llista",
```

Comment:
I believe "li" here is unnecessary, as you are already including the list in the sentence, so it is not replacing anything as a pronoun, in the end.